docs/runbook-trec2018-h2oloo.md

```diff
@@ -7,194 +7,206 @@ Note that this document is specifically a **runbook** and does not encode regres
+ -index lucene-index.core18.pos+docvectors+rawdocs -bm25 \
+ -topics src/main/resources/topics-and-qrels/topics.core18.txt \
+ -output run.core18.bm25+ax.topics.core18.txt -axiom \
+ -rerankCutoff 20 -axiom.deterministic -hits 10000
 ```
 
 ### Extraction of TF-IDF vectors
 
-- Robust04
+**Robust04**:
 
 ```bash
-target/appassembler/bin/IndexUtils -index lucene-index.robust04.pos+docvectors
--dumpAllDocids NONE
+target/appassembler/bin/IndexUtils -index lucene-index.robust04.pos+docvectors+rawdocs \
+ -dumpAllDocids NONE
 
-target/appassembler/bin/IndexUtils -index lucene-index.robust04.pos+docvectors
--dumpDocVectors lucene-index.robust04.pos+docvectors.allDocids.txt \
--docVectorWeight TF_IDF
+target/appassembler/bin/IndexUtils -index lucene-index.robust04.pos+docvectors+rawdocs \
+ -dumpDocVectors lucene-index.robust04.pos+docvectors+rawdocs.allDocids.txt \
+ -docVectorWeight TF_IDF
 ```
 
-- Robust05
+**Robust05**:
 
 ```bash
-target/appassembler/bin/IndexUtils -index lucene-index.robust05.pos+docvectors
--dumpAllDocids NONE
+target/appassembler/bin/IndexUtils -index lucene-index.robust05.pos+docvectors+rawdocs \
+ -dumpAllDocids NONE
 
-target/appassembler/bin/IndexUtils -index lucene-index.robust05.pos+docvectors
--dumpDocVectors lucene-index.robust05.pos+docvectors.allDocids.txt \
--docVectorWeight TF_IDF
+target/appassembler/bin/IndexUtils -index lucene-index.robust05.pos+docvectors+rawdocs \
+ -dumpDocVectors lucene-index.robust05.pos+docvectors+rawdocs.allDocids.txt \
+ -docVectorWeight TF_IDF
 ```
 
-- Core17
+**Core17**:
 
 ```bash
-target/appassembler/bin/IndexUtils -index lucene-index.core17.pos+docvectors
--dumpAllDocids NONE
+target/appassembler/bin/IndexUtils -index lucene-index.core17.pos+docvectors+rawdocs \
+ -dumpAllDocids NONE
 
-target/appassembler/bin/IndexUtils -index lucene-index.core17.pos+docvectors
--dumpDocVectors lucene-index.core17.pos+docvectors.allDocids.txt \
--docVectorWeight TF_IDF
+target/appassembler/bin/IndexUtils -index lucene-index.core17.pos+docvectors+rawdocs \
+ -dumpDocVectors lucene-index.core17.pos+docvectors+rawdocs.allDocids.txt \
+ -docVectorWeight TF_IDF
 ```
 
-- Core18
+**Core18**:
 
 ```bash
-target/appassembler/bin/IndexUtils -index lucene-index.wapo.pos+docvectors
--dumpAllDocids NONE
+target/appassembler/bin/IndexUtils -index lucene-index.core18.pos+docvectors+rawdocs \
+ -dumpAllDocids NONE
 
-target/appassembler/bin/IndexUtils -index lucene-index.wapo.pos+docvectors
--dumpDocVectors lucene-index.wapo.pos+docvectors.allDocids.txt \
--docVectorWeight TF_IDF
+target/appassembler/bin/IndexUtils -index lucene-index.core18.pos+docvectors+rawdocs \
+ -dumpDocVectors lucene-index.core18.pos+docvectors+rawdocs.allDocids.txt \
+ -docVectorWeight TF_IDF
 ```
 
-## Classifier Training/Inference
+## Classifier Training/Inference with Scikit-Learn
 
 Now we are done with Anserini part; everything below is in Python.
 
 Create the following structure folder structure:
```

Comment:
Typo here, maybe

Create a folder with the following structure

Reply:
fixed typo.